ref/general/authentication.adoc

```diff
@@ -34,18 +38,30 @@ Open Liberty supports the following options for SSO authentication:
 - link:/docs/ref/feature/#socialLogin-1.0.html[Social Media Login] +
 Users can authenticate to websites that are hosted on the Open Liberty server by logging in with their social media accounts. When the `socialLogin-1.0` feature is enabled, login with Facebook, Twitter, Google, Linkedin, and GitHub is automatically configured. You can also define your own configuration for any social media platform that is based on the OAuth 2.0 or OpenID Connect 1.0 standards.
 
+- link:/docs/ref/general/#/docs/concept/sso-config-json.html[JSON Web Tokens] +
+JSON Web Tokens (JWT) are used to propagate the user identity that is established by Single sign-on (SSO) between different microservice applications. By configuring SSO user authentication with JWT, you can manage the identity that is shared between applications.
+
 - link:/docs/ref/general/#/docs/concept/sso-config-ltpa.html[LTPA cookies] +
 Lightweight Third Party Authentication (LTPA) provides SSO configuration support to authenticate users when they are accessing application servers. With LTPA enabled, you can encrypt, digitally sign, and securely transmit authentication-related data between servers.
 
 - link:/docs/ref/feature/#openidConnectServer-1.0.html[OpenID Connect]  +
-OIDC is a popular internet SSO protocol that works well with cloud-native applications. This protocol enables a client application to request the identity of the user as an ID token in a standardized, REST-like manner. The client application, or Relying Party, requests information about a user from an authorization server, known as an OpenID Connect Provider. After users are authenticated by an OpenID Provider, they can access any Relying Party that is configured for that Provider, without having to manage unique accounts for each application or website.
+OIDC is a popular internet SSO protocol that works well with cloud-native applications. This protocol enables a client application to request the identity of the user as an ID token in a standardized, REST-like manner. The client application, or Relying Party, requests information about a user from an authorization server, which is known as an OpenID Connect Provider. After users are authenticated by an OpenID Provider, they can access any Relying Party that is configured for that Provider, without having to manage unique accounts for each application or website.
```

Comment:
Is this first reference to OIDC? If so should the meaning be spelled out first? Not sure what changed here.

Reply:
not sure why there's a dif here but I added the acronym to the heading so it's defined for the description..